internal/pkg/reconcilers/bucket/reconciler.go

```diff
@@ -0,0 +1,13 @@
+package bucket
+
+import (
+	"context"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"github.com/giantswarm/object-storage-operator/api/v1alpha1"
+)
+
+type BucketReconciler interface {
```

Comment:
What's the reason for this interface? Why does `CAPABucketReconciler` need to be a reconciler if it's not going to be managed by the controller manager?

Reply:
I think it's because the name made sense to me but I'll make it a service